kolibri/plugins/example_plugin/templates/kolibri/plugins/example_plugin/index.html

```diff
@@ -0,0 +1,5 @@
+{% extends "kolibri/base.html" %}
+
+{% block content %}
+This is an example
+{% endblock %}
```

Comment:
Might also be worth adding an example 'snippets' or 'partials' directory, as plugins may be plugging into other templates?